RELEASE-NOTES.md

```diff
@@ -1,21 +1,25 @@
 # About this release
 
-[This release](https://github.com/aeternity/epoch/releases/tag/v0.4.1) is focused on stability of the testnet - the public test network of nodes and introduces some API for apps usability build on top of it.
-In this release the [Oracle WebSocket API](https://github.com/aeternity/protocol/blob/epoch-v0.5.0/epoch_api/oracle_api.md) is now documented and can be used to interact with Oracles on the block chain
-(See also [Oracles - intended usage](https://github.com/aeternity/protocol/blob/epoch-v0.5.0/oracles/oracle_ws_api.md).
+[This release](https://github.com/aeternity/epoch/releases/tag/v0.5.0) is focused on oracles: the [Oracle WebSocket API](https://github.com/aeternity/protocol/blob/epoch-v0.5.0/epoch_api/oracle_ws_api.md) is documented and can be used to interact with Oracles on the block chain (see also [Oracles - intended usage](https://github.com/aeternity/protocol/blob/epoch-v0.5.0/epoch_api/oracle_api_usage.md)).
```

Comment:
@hanssv I updated this section to recent https://github.com/aeternity/protocol repo